website/static/js/registrationUtils.js

```diff
@@ -779,7 +781,54 @@ RegistrationEditor.prototype.save = function() {
     }
     return true;
 };
+/**
+ * Returns the `user_fullname` of each contributor attached to a node.
+ **/
+RegistrationEditor.prototype.getContributors = function() {
+    var contributorsUrl = window.contextVars.node.urls.api + 'contributors_abbrev/';
+    var contributorNames = [];
+    $osf.ajaxJSON('get', contributorsUrl).done(function(data) {
+        $.each(data.contributors, function(idx, val) {
+            contributorNames.push(val.user_fullname);
```

Comment:
Consider using `jquery.map` instead of `.each`.
http://api.jquery.com/jquery.map/